website/docs/migration.md

```diff
@@ -0,0 +1,216 @@
+---
+sidebar_position: 9
+---
+
+# Migrating from MongoDB to FerretDB
+
+To ensure a smooth and successful migration from MongoDB, we offer several methods through which you can test your application with FerretDB.
+
+## Operation modes
+
+We offer multiple operation modes which help facilitate the testing of your application by enabling FerretDB to act as a proxy.
+For more details, refer to the [official documentation](configuration/operation-modes.md).
```

Comment:
"official" sounds strange there – we are just referring to another documentation page. They are both "official"

Reply:
I agree - fixed